FirebaseInstallations/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-# v1.0.1 -- M62
+# v1.1.0 -- M62.1
 
-- [changed] Throw an exception when there are missing required `FIROptions` parameters. (#4683)
+- [changed] Throw an exception when there are missing required `FirebaseOptions` parameters (`APIKey`, `googleAppID`, and `projectID` or `GCMSenderID`). Please make sure your `GoogleServices-Info.plist` (or `FirebaseOptions` if you configure Firebase in code) are up to date. The file and settings can be downoaded from the [Firebase Console](https://console.firebase.google.com/).  (#4683)
```

Comment:
nit: Throws  (with trailing "s")

I recommend to remove "GCMSenderID" and would not even mention it.
I think it has more potential to create confusion rather than help the developer understand what is happening.
If the developer changes something, we want him to add Project-ID.

Question: Is it called FirebaseOptions or FIROptions?